test/Client/Grpc.Tests/DependencyInjection/DurableTaskClientBuilderExtensionsTests.cs

```diff
@@ -67,6 +67,20 @@ public void UseGrpc_Callback_Sets()
         options.Address.Should().BeNull();
     }
 
+    [Fact]
+    public void UseDefaultVersion_DefaultVersion_Sets()
+    {
+        ServiceCollection services = new();
+        DefaultDurableTaskClientBuilder builder = new(null, services);
+        builder.UseDefaultVersion("0.1")
+            .UseGrpc();
+
+        var provider = services.BuildServiceProvider();
+        var options = provider.GetOptions<GrpcDurableTaskClientOptions>();
```

Comment:
nit: Same comment about the use of `var`. It would be best if we could keep the code style consistent when possible (see method above where we use explicit type names).

Reply:
Done.